src/App.tsx

```diff
@@ -1,9 +1,75 @@
 import './App.scss';
+import {
+  Navigate, NavLink, Route, Routes,
+} from 'react-router-dom';
+import { useEffect, useState } from 'react';
+import { getPeople } from './api';
+import { Person } from './types/Person';
+import { PeopleTable } from './components/Peopletable/PeopleTable';
+import { PersonInfo } from './components/Personinfo/Personinfo';
+import { Page404 } from './components/Page404/Page404';
 
-const App = () => (
-  <div className="App">
-    <h1>People table</h1>
-  </div>
-);
+export const App: React.FC = () => {
+  const [people, setPeople] = useState<Person[]>([]);
 
-export default App;
+  useEffect((() => {
+    getPeople().then(res => {
+      if (res) {
+        setPeople(res);
+      }
+    });
+  }), []);
```

Comment:
move it to `PeopleTable` component, you don't share it with other components, whatsoever